data/json/professions.json

```diff
@@ -5138,5 +5138,145 @@
                 "panties"
             ]
         }
+    },
+    {
+        "type": "profession",
+        "ident": "escape_con",
+        "name": "Escaped Prisioner",
+        "description": "When the cataclysm started you took your chance, now you roam looking for a place to stay. Hopefully those days workin in prison pay off..",
+        "points": -2,
+        "skills": [
+            {
+                "level": 1,
+                "name": "mechanics"
+            },
+            {
+                "level": 3,
+                "name": "fabrication"
+            }
+        ],
+        "items": {
+            "both": [
+                "technician_shirt_gray",
+                "striped_pants",
+                "greatcoat",
+                "manual_knives",
+                "knife_steak",
+                "gloves_fingerless"
+            ],
+            "male": [
+                "boxer_shorts"
+            ],
+            "female": [
+                "bra",
+                "panties"
+            ]
+        }
+    },
+    {
+        "type": "profession",
+        "ident": "bad_burglar",
+        "name": "Amateur Burglar",
+        "description": "You thought life'd be better after the catclysm. Oh, were you wrong.",
+        "points": 2,
+        "items": {
+            "both": [
+                "socks",
+                "boots",
+                "pants",
+                "tshirt",
+                "hoodie",
+                "gloves_light",
+                "balclava",
+                "swag_bag",
+                "makeshift_crowbar",
+                "picklocks"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "bra",
+                "panties"
+            ]
+        }
+    },
+    {
+        "type": "profession",
+        "ident": "bio_sci",
+        "name": "Bionics Scientist",
+        "description": "Before all this you studied bioncs. You tried your experiments on animals and yourself.",
+        "points": 4,
```

Comment:
Needs a justification for the cost. All classes with bionics or traits need one by default.

Reply:
?  I tried to level it with the Animal trait.